qiskit/providers/ibmq/credentials/environ.py:19:0: W0611: Unused Tuple imported from typing (unused-import)

### Information

- **Qiskit IBMQ Provider version**: master
- **Python version**: 3.7
- **Operating system**: n/a

### What is the current behavior?

CI runs are failing the lint check on this:

```
Run make style && make lint
pycodestyle qiskit test
pylint -rn qiskit/providers/ibmq test
************* Module ibmq.credentials.environ
qiskit/providers/ibmq/credentials/environ.py:19:0: W0611: Unused Tuple imported from typing (unused-import)
```

E.g. https://github.com/Qiskit/qiskit-ibmq-provider/pull/829/checks?check_run_id=1512969962


### Steps to reproduce the problem

Any PR that runs through CI is hitting this, see #829, #830, #831.

### What is the expected behavior?

CI should not fail on unrelated changes.

### Suggested solutions

The lint failure is legitimate, it's just curious why it started all of a sudden.

--- a/qiskit/providers/ibmq/credentials/environ.py
+++ b/qiskit/providers/ibmq/credentials/environ.py
@@ -16,7 +16,7 @@
 
 import os
 from collections import OrderedDict
-from typing import Dict, Tuple
+from typing import Dict
 
 from .credentials import Credentials
 from .hubgroupproject import HubGroupProject

